pkg/customizations/firstboot/firstboot.go

```diff
@@ -0,0 +1,91 @@
+package firstboot
+
+import (
+	"errors"
+	"net/url"
+
+	"github.com/osbuild/images/pkg/blueprint"
+)
+
+type FirstbootOptions struct {
+	Custom    []CustomFirstbootOptions
+	Satellite *SatelliteFirstbootOptions
+	AAP       *AAPFirstbootOptions
+}
+
+type CustomFirstbootOptions struct {
+	Contents      string
+	Name          string
+	IgnoreFailure bool
+}
+
+type SatelliteFirstbootOptions struct {
+	Command string
+	CACerts []string
+}
+
+type AAPFirstbootOptions struct {
+	JobTemplateURL string
+	HostConfigKey  string
+	CACerts        []string
```

Comment:
I think CRC also accepts a skip tls verification option too:

https://github.com/osbuild/image-builder-crc/blob/cbfac18e1a143fc761a396a4b2a0b3538dba2fb1/internal/v1/api.yaml#L1480-L1492

Reply:
I was thinking not having any certs would skip but being explicit is probably safer.

Reply:
Ya that's valid, just making a note of it :)